projects/common/src/index.ts

```diff
@@ -301,6 +300,10 @@ export interface Image {
     source: string
     path: string
 }
+
+export interface CreditedImage extends Image {
```

Comment:
Wouldnt this just be an optional parameter to Image rather than a new interface?

Reply:
It could be but I sort of wanted to specify the difference in the typing for readability. An `Image` doesn't have a credit because it doesn't need to be specified in the context, a `CreditedImage` would not have a credit if it couldn't find one but wanted one.